[Terraform] Fix examples (#8911)
cloudflare/cloudflare-docs: content/terraform/advanced-topics/import-cloudflare-resources.md

      + modified_on = (known after apply)
      + name        = "a123.mitigateddos.net"
      + proxiable   = (known after apply)
      + proxied     = false
      + ttl         = 300
      + type        = "NS"
      + value       = "rafe.ns.cloudflare.com"
      + zone_id     = "1109d899a5ff5fd74bc01e581693685b"
    }

  # cloudflare_record.terraform_managed_resource_5799bb01054843eea726758f935d2aa2 will be created
  + resource "cloudflare_record" "terraform_managed_resource_5799bb01054843eea726758f935d2aa2" {
      + id          = (known after apply)
      + created_on  = (known after apply)
      + domain      = "mitigateddos.net"
      + hostname    = (known after apply)
      + metadata    = (known after apply)
      + modified_on = (known after apply)
      + name        = "a123.mitigateddos.net"
      + proxiable   = (known after apply)
      + proxied     = false
      + ttl         = 300
      + type        = "NS"
      + value       = "terin.ns.cloudflare.com"
      + zone_id     = "1109d899a5ff5fd74bc01e581693685b"
    }

Plan: 4 to add, 0 to change, 0 to destroy.

------------------------------------------------------------------------

> Note: You didn't use the -out option to save this plan, so Terraform can't

Reviewer: ```suggestion
Note: You did not use the `-out` option to save this plan, so Terraform cannot
```

Author: I don't think we should change this part, since this text is in the tool output as is.

Author: (Also part of the tool output.)

> guarantee to take exactly these actions if you run "terraform apply" now.

Reviewer: ```suggestion
guarantee to take exactly these actions if you run `terraform apply` now.
```
